src/platform/user/profile/vap-svc/actions/ui.js

```diff
@@ -36,7 +36,10 @@ export const updateFormFieldWithSchema = (
     true,
   );
 
-  if (options.doNotRequireInternationalPostalCode) {
+  if (
```

Comment:
It was trying to set an unavailable property and was causing other fields to not show in edit mode